sdk/KoreBuild/modules/frameworkbuild/3-compile.targets

```diff
@@ -0,0 +1,75 @@
+<Project>
+  <PropertyGroup>
+  	<CompileDependsOn>$(CompileDependsOn);CompileProjects;CompileRuntimeStore</CompileDependsOn>
```

Comment:
I'm assuming `CompileRuntimeStore` builds the individual runtime stores for each RID of the platforms we support. I wouldn't include it in the default compile targets of `CompileProjects` as this will need to be run on 4 different agents. Keep it as a separate target that's not in the default CompileDependsOn property.

Reply:
That's a good point. We should decouple CompileRuntimeStore from CompileProjects. You should be able to copmile the runtime store without re-compiling aspnetcore from sourc.e

FYI - my thinking was that you should be able to call `build /t:Compile` and it should compile all aspnetcore source and the runtime store. Likewise, `build /t:Package` would produce all nupkgs, runtime stores, installers, and any other kind of artifact that we ship as a "package" of one form or another.